projects/WTI-UI/src/app/modules/clarifications/components/new-announcement-clarification-alert/new-announcement-alert.component.html

```diff
@@ -0,0 +1,9 @@
+<h1 mat-dialog-title>Clarification Announcement Received</h1>
```

Comment:
I would suggest the title here be just "Announcement Received", in order to avoid confusion between what's a "clarification" vs. what's an "announcement".  (PC2 is, currently, treating "announcements" as "clarifications", but it seems to make more sense from the user's perspective to keep the two notions separate.)

Reply:
For similar reasons I think it would make sense to rename all of the components which are named like `new-announcement-clarification-alert` to be just `new-announcement-alert`.  It would simplify the naming and at the same time help maintain the distinction between "announcements" and "clarifications".

Reply:
The reason I say clarification announcement is to emphasize the fact that announcements are essentially a special form of clarification. The term "announcement clarification" was decided when I was building this feature in pc2. Of course that can be discussed and changed.